pyproject.toml

```diff
@@ -59,7 +60,10 @@ disable = [
     "missing-class-docstring",
     "missing-function-docstring",
     "missing-module-docstring",
-    "too-few-public-methods"
+    "too-few-public-methods",
+    "logging-not-lazy",
+    "consider-using-f-string",
+    "logging-fstring-interpolation"
```

Comment:
[question]
Are these disables needed? It's an easy fix on the logging and follows our playbook. (See this comment)

Reply:
He complains about anything you use.
The entire lint configuration is compromised in my view. I already removed the `tox` to speed up the tests, I'm going to go back to the lib and redo all the correct lint configuration.